openstack/sap-seeds/templates/flavor-seed.yaml

```diff
@@ -267,6 +267,15 @@ spec:
     is_public: true
     extra_specs:
       {{- tuple . "vmware" | include "sap_seeds.helpers.extra_specs" | indent 6 }}
+  - name: "m_c2_m16"
+    id: "138"
+    vcpus: 2
+    ram: 16368
+    disk: 64
+    is_public: true
+    extra_specs:
+      "catalog:alias": "m2.4xlarge"
```

Comment:
This seems to be an error during copy and paste as we now have 2 flavors with the same alias.

Reply:
oops, thanks!